### ISSUE TITLE
coverage data in dist folder files of npm package interfering with jest coverage

### ISSUE THREAD

**Issue by paradite** [vinothpandian/react-sketch-canvas#63]
**Describe the bug**
Coverage data are found in various files in dist folder of the npm package, causing increased file size and more importantly interfering with jest coverage data (running jest with coverage reports extra info from this pacakge).

**To Reproduce**
1. Go to https://unpkg.com/react-sketch-canvas@6.1.0/dist/react-sketch-canvas.cjs.development.js and check the file content
2. Go to https://unpkg.com/react-sketch-canvas@6.1.0/dist/react-sketch-canvas.esm.js and check the file content
3. Go to https://unpkg.com/react-sketch-canvas@6.1.0/dist/react-sketch-canvas.cjs.production.min.js and check the file content

Observe the following:
```js
function cov_20wqqco38b() {
  var path = "/home/vinoth/projects/react-sketch-canvas/src/Paths/index.tsx";
  var hash = "80c4a162af9ff11419db7b7c460ec352bdae34fe";
  var global = new Function("return this")();
  var gcv = "__coverage__";
  var coverageData = {
    path: "/home/vinoth/projects/react-sketch-canvas/src/Paths/index.tsx",
    statementMap: {
      "0": {
        start: {
          line: 5,
          column: 23
        },
        end: {
          line: 13,
          column: 1
        }
      },
      "1": {
        start: {
          line: 6,
          column: 4
        },
        end: {
          line: 10,
          column: 5
        }
      },
      "2": {
        start: {
          line: 7,
          column: 25
        },
        end: {
          line: 7,
          column: 33
...
```

**Expected behavior**
no coverage data inside

**Screenshots**
file content:
<img width="913" alt="Screenshot 2022-02-04 at 10 08 28 PM" src="https://user-images.githubusercontent.com/1209810/152542805-c9d1542e-adaa-4e10-9990-bbea45c460b2.png">

production min file:
<img width="1425" alt="Screenshot 2022-02-04 at 10 16 06 PM" src="https://user-images.githubusercontent.com/1209810/152544185-19e7bd1b-6714-429d-8098-cb6bd5d07e95.png">

interference with jest:
<img width="1098" alt="Screenshot 2022-02-04 at 10 11 13 PM" src="https://user-images.githubusercontent.com/1209810/152543371-5e4144fb-456f-4aaf-ae48-b4279dc7e420.png">

**Desktop (please complete the following information):**
-

**Smartphone (please complete the following information):**
-

**Additional context**
-

**Comment by vinothpandian** [comment]
Fixed in 6.1.5

**Comment by RosenTomov** [comment]
Looks like this issue was reintroduced in the pre-release version 7.0.0-next.2

**Comment by vinothpandian** [comment]
Thanks for notifying me. I'll create an issue

**Comment by vinothpandian** [comment]
Fixed it in 7.0.0-next.3
